src/lib/components/chat/ChatWindow.svelte

```diff
@@ -226,12 +246,19 @@
 				</p>
 				{#if messages.length}
 					<button
-						class="flex flex-none items-center hover:text-gray-400 hover:underline max-sm:rounded-lg max-sm:bg-gray-50 max-sm:px-2.5 dark:max-sm:bg-gray-800"
+						class="flex flex-none items-center hover:text-gray-400 max-sm:rounded-lg max-sm:bg-gray-50 max-sm:px-2.5 dark:max-sm:bg-gray-800"
 						type="button"
+						class:hover:underline={!$isConversationShared}
 						on:click={() => dispatch("share")}
+						disabled={$isConversationShared}
 					>
-						<CarbonExport class="text-[.6rem] sm:mr-1.5 sm:text-primary-500" />
-						<div class="max-sm:hidden">Share this conversation</div>
+						{#if $isConversationShared}
```

Comment:
```suggestion
						{#if isSharedRecently}
```

Reply:
handled in https://github.com/huggingface/chat-ui/pull/645/commits/fdbb4cafbf846c95557cb7e2fe0ce45effcd4c76